src/boi/boi_server.cpp

```diff
@@ -11,29 +11,32 @@ boi_wifi::WifiModeEnum Mode;
 
 RequestHandler::RequestHandler() {}
 
-bool RequestHandler::canHandle(AsyncWebServerRequest *request){
-    return true;
+bool RequestHandler::canHandle(AsyncWebServerRequest *request) { // this method is our config to tell ESPAsyncWebserver what requests we want to handle
```

Comment:
This is an incomplete feature from our addition of the batteryinter.net/connect feature for connecting to the device over your home network.